src/api/runtime.ts

```diff
@@ -32,13 +29,8 @@ export function makeRuntime({
   requireModules: string[]
   requirePaths: string[]
   importPaths: string[]
-  options: Partial<IRuntimeOptions> & { parallel?: number }
+  options: IRunOptionsRuntime
 }): IRuntime {
-  // sprinkle specified runtime options over the defaults
```

Comment:
No longer needed here because defaulting is done as part of loading/merging/validating configuration.